gorp.go

```diff
@@ -114,6 +114,14 @@ type SqlExecutor interface {
 	queryRow(query string, args ...interface{}) *sql.Row
 }
 
+// DynamicTable allows the users of gorp to dynamically
+// use different database table names during runtime
+// while sharing the same golang struct for in-memory data
+type DynamicTable interface {
+	GetTableName() string
```

Comment:
Sure, I can change it to TableName(), will update the code within the next couple of days. Thanks!
Biju